### PR TITLE
rnashapes: add linux-aarch64 support

### PR DESCRIPTION
**Pull request by martin-g**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

### General instructions

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

### Instructions for avoiding API, ABI, and CLI breakage issues
Conda is able to record and lock (a.k.a. pin) dependency versions used at build time of other recipes.
This way, one can avoid that expectations of a downstream recipe with regards to API, ABI, or CLI are violated by later changes in the recipe.
If not already present in the meta.yaml, make sure to specify `run_exports` (see [here](https://bioconda.github.io/contributor/linting.html#missing-run-exports) for the rationale and comprehensive explanation).
Add a `run_exports` section like this:

```yaml
build:
  run_exports:
    - ...

```

with `...` being one of:

| Case                             | run_exports statement                                               |
| -------------------------------- | ------------------------------------------------------------------- |
| semantic versioning              | `{{ pin_subpackage("myrecipe", max_pin="x") }}`     |
| semantic versioning (0.x.x)      | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}`   |
| known breakage in minor versions | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| known breakage in patch versions | `{{ pin_subpackage("myrecipe", max_pin="x.x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| calendar versioning              | `{{ pin_subpackage("myrecipe", max_pin=None) }}`    |

while replacing `"myrecipe"` with either `name` if a `name|lower` variable is defined in your recipe or with the lowercase name of the package in quotes.

### Bot commands for PR management

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

Note that the <code>@BiocondaBot please merge</code> command is now depreciated. Please just squash and merge instead.

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/rnashapes/meta.yaml [hunk 1]
```diff
@@ -50,5 +50,8 @@ about:
     account of properties of the molecule's folding space."
 
 extra:
+  additional-platforms:
+    - linux-aarch64
+    - osx-arm64
```
Comment (martin-g) [inline recipes/rnashapes/meta.yaml:55]:
```suggestion
```

**Comment by martin-g** [comment]
@BiocondaBot please fetch artifacts

**Comment by martin-g** [comment]
```
mgrigorov in 🌐 euler-arm-22 in /tmp/rnashapes via 🅒 test123 took 5s 
❯ tar xvf rnashapes-3.4.0-pl5321h78569d1_1.tar.bz2                                                                                                                                                         (test123) 
info/run_exports.json
info/test/run_test.sh
info/hash_input.json
info/recipe/build.sh
info/has_prefix
info/index.json
info/recipe/meta.yaml.template
info/files
info/recipe/meta.yaml
info/recipe/conda_build_config.yaml
info/about.json
info/paths.json
info/licenses/LICENSE
info/git
bin/RNAshapes
bin/RNAshapes_pfall_nodangle
bin/RNAshapes_pfall_overdangle
bin/RNAshapes_pfall_nodangle_window
bin/RNAshapes_pfall_overdangle_window
bin/RNAshapes_pfall_microstate
bin/RNAshapes_pfall_microstate_window
bin/RNAshapes_outside_nodangle
bin/RNAshapes_outside_overdangle
bin/RNAshapes_outside_microstate
bin/RNAshapes_pfall_macrostate
bin/RNAshapes_pfall_macrostate_window
bin/RNAshapes_eval_nodangle
bin/RNAshapes_eval_overdangle
bin/RNAshapes_eval_microstate
bin/RNAshapes_probing_nodangle
bin/RNAshapes_probing_nodangle_window
bin/RNAshapes_probing_overdangle
bin/RNAshapes_probing_overdangle_window
bin/RNAshapes_probing_microstate
bin/RNAshapes_probing_microstate_window
bin/RNAshapes_eval_macrostate
bin/RNAshapes_mea_nodangle_window
bin/RNAshapes_mea_overdangle_window
bin/RNAshapes_mea_nodangle
bin/RNAshapes_mea_overdangle
bin/RNAshapes_sample_nodangle_window
bin/RNAshapes_sample_overdangle_window
bin/RNAshapes_sample_nodangle
bin/RNAshapes_sample_overdangle
bin/RNAshapes_mfe_nodangle_window
bin/RNAshapes_mfe_overdangle_window
bin/RNAshapes_mfe_nodangle
bin/RNAshapes_mfe_overdangle
bin/RNAshapes_mea_microstate_window
bin/RNAshapes_mea_microstate
bin/RNAshapes_sample_microstate_window
bin/RNAshapes_sample_microstate
bin/RNAshapes_shapes_nodangle_window
bin/RNAshapes_shapes_overdangle_window
bin/RNAshapes_shapes_nodangle
bin/RNAshapes_shapes_overdangle
bin/RNAshapes_probs_nodangle_window
bin/RNAshapes_probs_nodangle
bin/RNAshapes_probs_overdangle_window
bin/RNAshapes_probs_overdangle
bin/RNAshapes_subopt_nodangle_window
bin/RNAshapes_subopt_overdangle_window
bin/RNAshapes_subopt_nodangle
bin/RNAshapes_subopt_overdangle
bin/RNAshapes_mfe_microstate_window
bin/RNAshapes_mfe_microstate
bin/RNAshapes_probing_macrostate_window
bin/RNAshapes_probing_macrostate
bin/RNAshapes_shapes_microstate_window
bin/RNAshapes_shapes_microstate
bin/RNAshapes_probs_microstate_window
bin/RNAshapes_probs_microstate
bin/RNAshapes_subopt_microstate_window
bin/RNAshapes_subopt_microstate
bin/RNAshapes_sample_macrostate_window
bin/RNAshapes_sample_macrostate
bin/RNAshapes_mfe_macrostate_window
bin/RNAshapes_mfe_macrostate
bin/RNAshapes_shapes_macrostate_window
bin/RNAshapes_shapes_macrostate
bin/RNAshapes_subopt_macrostate_window
bin/RNAshapes_subopt_macrostate
bin/RNAshapes_probs_macrostate_window
bin/RNAshapes_probs_macrostate
lib/foldGrammars/ClusterSettings.pm
lib/foldGrammars/CMs.pm
lib/foldGrammars/NewCollection.pm
lib/foldGrammars/Settings.pm
lib/foldGrammars/RNAcast.pm
lib/foldGrammars/References.pm
lib/foldGrammars/RapidShapes.pm
lib/foldGrammars/Structure.pm
lib/foldGrammars/Testing.pm
lib/foldGrammars/Utils.pm
lib/foldGrammars/IO.pm
bin/addRNAoptions.pl

mgrigorov in 🌐 euler-arm-22 in /tmp/rnashapes via 🅒 test123 
❯ file bin/*                                                                                                                                                                                               (test123) 
bin/addRNAoptions.pl:                    Perl script text executable
bin/RNAshapes:                           Perl script text executable
bin/RNAshapes_eval_macrostate:           ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_eval_microstate:           ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_eval_nodangle:             ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_eval_overdangle:           ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_mea_microstate:            ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_mea_microstate_window:     ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_mea_nodangle:              ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_mea_nodangle_window:       ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_mea_overdangle:            ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_mea_overdangle_window:     ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_mfe_macrostate:            ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_mfe_macrostate_window:     ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_mfe_microstate:            ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_mfe_microstate_window:     ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_mfe_nodangle:              ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_mfe_nodangle_window:       ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_mfe_overdangle:            ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_mfe_overdangle_window:     ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_outside_microstate:        ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, not stripped
bin/RNAshapes_outside_nodangle:          ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, not stripped
bin/RNAshapes_outside_overdangle:        ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, not stripped
bin/RNAshapes_pfall_macrostate:          ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, not stripped
bin/RNAshapes_pfall_macrostate_window:   ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, not stripped
bin/RNAshapes_pfall_microstate:          ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, not stripped
bin/RNAshapes_pfall_microstate_window:   ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, not stripped
bin/RNAshapes_pfall_nodangle:            ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, not stripped
bin/RNAshapes_pfall_nodangle_window:     ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, not stripped
bin/RNAshapes_pfall_overdangle:          ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, not stripped
bin/RNAshapes_pfall_overdangle_window:   ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, not stripped
bin/RNAshapes_probing_macrostate:        ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_probing_macrostate_window: ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_probing_microstate:        ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_probing_microstate_window: ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_probing_nodangle:          ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_probing_nodangle_window:   ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_probing_overdangle:        ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_probing_overdangle_window: ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_probs_macrostate:          ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_probs_macrostate_window:   ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_probs_microstate:          ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_probs_microstate_window:   ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_probs_nodangle:            ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_probs_nodangle_window:     ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_probs_overdangle:          ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_probs_overdangle_window:   ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_sample_macrostate:         ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_sample_macrostate_window:  ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_sample_microstate:         ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_sample_microstate_window:  ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_sample_nodangle:           ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_sample_nodangle_window:    ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_sample_overdangle:         ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_sample_overdangle_window:  ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_shapes_macrostate:         ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_shapes_macrostate_window:  ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_shapes_microstate:         ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_shapes_microstate_window:  ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_shapes_nodangle:           ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_shapes_nodangle_window:    ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_shapes_overdangle:         ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_shapes_overdangle_window:  ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_subopt_macrostate:         ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_subopt_macrostate_window:  ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_subopt_microstate:         ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_subopt_microstate_window:  ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_subopt_nodangle:           ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_subopt_nodangle_window:    ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_subopt_overdangle:         ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped
bin/RNAshapes_subopt_overdangle_window:  ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, with debug_info, not stripped

mgrigorov in 🌐 euler-arm-22 in /tmp/rnashapes via 🅒 test123 
❯ ./bin/RNAshapes                                                                                                                                                                                          (test123) 
You are in "shapes" mode. Please give me either your RNA sequence, or a (multiple) FASTA file, containing your sequences:
^C⏎                                                                                                                                                                                                                  
```
LGTM!

**Comment by martin-g** [comment]
@BiocondaBot please add label

**Comment by sjanssen2** [comment]
I very much like the idea to make RNAshapes available on aarch64, however, I am not sure if this will run out of the box. RNAshapes is a Perl wrapper around multiple compiled binaries. These binaries are the result of a GAP-L to C++ transpilation with subsequent g++ compilation. The test command in this recipe "just" tests if the Perl wrapper is available - there are no functional tests of the binaries. We would somehow ensure that results are identical. A possible route would be:
  1.  extend tests / actions of https://github.com/jlab/gapc to aarch64 (the transpiler)
  2.  same for https://github.com/jlab/fold-grammars (instructions for binaries)
  3. publish bioconda packages

**Pull request merged**